modules/templates.js

```diff
@@ -0,0 +1,13 @@
+angular.module('pages').run(['$templateCache', function($templateCache) {
```

Comment:
> why ? our other repos have the templates.js file outside the folders. so i dont think so. For consistancy it should follow our other repos

@Minglee01 They usually have them within each module, so the `feincms-pages` module should have it's own `templates.js`, and the `foo` module, like
- `modules/foo/templates.js`
- `modules/bar/templates.js`

Otherwise `modules/templates.js` is one templates file for all modules. If another module is added, which module takes precedence for this one file?